pkg/apis/core/v1/defaults.go

```diff
@@ -180,7 +181,7 @@ func SetDefaults_Pod(obj *v1.Pod) {
 				}
 			}
 		}
-		if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) &&
+		if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.InPlacePodVerticalScaling) &&
```

Comment:
ahh test failure looks to be from an import cycle. Let me think about this a bit more.